e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -137,4 +138,11 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('M23361 Focus remains in the RHS text box', () => {
+        cy.postMessage(MESSAGES.MEDIUM);
```

Comment:
Please add comments like I mentioned in your other PR - https://github.com/mattermost/mattermost-webapp/pull/5298/files#r408158174.

Reply:
Could I assume that the two-square channel contains at least one previous message?

Reply:
If you meant the town-square channels, yes, it's a safe assumption.